apps/console/src/features/users/components/wizard/add-user-wizard.tsx

```diff
@@ -201,15 +297,145 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
         }
     }, [ wizardState && wizardState[ WizardStepsFormTypes.BASIC_DETAILS ]?.domain ]);
 
-    const getGroupListForDomain = (domain: string) => {        
-        getGroupList(domain)
+    useEffect(() => {
+
+        if (!wizardState) {
+            return;
+        }
+
+        if (!fixedGroupList) {
+            return;
+        }
+
+        if (wizardState[ WizardStepsFormTypes.USER_TYPE ].userType === UserAccountTypes.USER) {
+            if (fixedGroupList?.length === 0) {
+                if (isUserSummaryEnabled) {
+                    setWizardSteps(filterSteps([
+                        WizardStepsFormTypes.BASIC_DETAILS,
+                        WizardStepsFormTypes.USER_SUMMARY
+                    ]));
+                    setIsStepsUpdated(true);
+
+                    return;
+                }
+                setWizardSteps(filterSteps([
+                    // Temporarily disable the USER_TYPE step.
```

Comment:
If this is temporary, let's create an issue in product IS and link here, so we don't miss it.

```suggestion
                    // TODO: Fix this once ..... Tracker: <product-is-issue>
                    // Temporarily disable the USER_TYPE step.
```